core/controllers/blog_homepage_test.py

```diff
@@ -18,17 +18,26 @@
 import logging
 
 from core import feconf
+from core.constants import constants
 from core.domain import blog_services
 from core.domain import config_domain
 from core.platform import models
 from core.tests import test_utils
 
+from typing import Final
+
 MYPY = False
 if MYPY:  # pragma: no cover
     from mypy_imports import user_models
 
 (user_models,) = models.Registry.import_models([models.Names.USER])
 
+MAX_CHARS_IN_BLOG_POST_URL: Final = (
+    constants.MAX_CHARS_IN_BLOG_POST_TITLE
+    + len('-')
+    + constants.BLOG_POST_ID_LENGTH
+)
+
```

Comment:
If `MAX_CHARS_IN_BLOG_POST_URL ` is the same value from `blog_hompage.py` can't we use that?

or maybe swap it to a lower value for the test

Reply:
we cannot import constant defined in blog_homepage.py file into blog_homepage_test file. Also swapping it directly with a lower value might lead to errors in future if value of `MAX_CHARS_IN_BLOG_POST_TITLE` or `BLOG_POST_ID_LENGTH` are changed in `constants` file, since it a calculated value and this might allow bugs to get in.

Reply:
I am not sure if swapping `MAX_CHARS_IN_BLOG_POST_URL` to a lower value would cause any errors in the future. Say in the future the values of the constants change, how would it affect the value over here?

By swapping, in the test, we are just saying any value > `MAX_CHARS_IN_BLOG_POST_URL` would fail.

Tomorrow if `MAX_CHARS_IN_BLOG_POST_URL` is changed in `blog_hompage.py`, we would have to change in the test file accordingly as well so for maintenance it would be better to use the same value initialized over there.

Reply:
Done. Made a single constant in feconf.py file, so that it can be imported from there and used directly instead of defining it again and again in files (It will also be required in the new files that will be added in the project)